packages/inventory/src/AppInfo.js

```diff
@@ -19,9 +27,18 @@ class AppInfo extends Component {
 }
 
 AppInfo.propTypes = {
+    store: PropTypes.any,
+    entity: PropTypes.shape({
+        id: PropTypes.string
+    }),
+    componentMapper: PropTypes.elementType,
```

Comment:
I just thought of this. The element type is available only in recent versions of prop types. I would remove prop types from externals. It will be deduped and all apps should have prop types anyway